lib/dockerize_rails/docker_commands/docker_stop.rb

```diff
@@ -0,0 +1,43 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerStop
+      def self.stop_rails
+        docker_stop DockerHelpers.get_name(:rails, :container)
+      end
+
+      def self.stop_mysql
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'mysql'
+          return docker_stop DockerHelpers.get_name(:mysql, :container)
+        end
+        0
+      end
+
+      def self.stop_postgres
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'postgresql'
+          return docker_stop DockerHelpers.get_name(:postgres, :container)
+        end
+        0
+      end
+
+      def self.docker_stop(container_name)
+        container = Docker::Container.get(container_name)
+        if container.info['State']['Running']
+          container.stop
+          container.delete(:force => true) if DRNameSpace.namespace.delete_containers
```

Comment:
__[rubocop]__
* Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)

<!-- Comment by [SideCI](https://sideci.com) -->